packages/@aws-cdk/aws-s3/lib/bucket.ts

```diff
@@ -1012,6 +1012,14 @@ export interface BucketProps {
    */
   readonly encryptionKey?: kms.IKey;
 
+  /**
+  * Enforces SSL for requests. S3.5 of the AWS Foundational Security Best Practices Regarding S3.
+  * @see https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-standards-fsbp-controls.html#fsbp-s3-5
```

Comment:
replace with the docs specific to s3: https://docs.aws.amazon.com/config/latest/developerguide/s3-bucket-ssl-requests-only.html

Reply:
@NetaNir Hi! I just updated it. Thanks so much!